_episodes/08-defensive.md

```diff
@@ -531,11 +531,13 @@ This violates another important rule of programming:
 > >
 > > def range_overlap(ranges):
 > >     '''Return common overlap among a set of [low, high] ranges.'''
+> >     if not ranges:
```

Comment:
We talk about `None` a bit before this exercise:
```
`None` is built into Python, and means "nothing here".
```

What about I add a comment:
```python
    if not ranges:
        # ranges is None or an empty list
        return None
```